AppKit/CPWindow/CPWindow.j

```diff
@@ -1958,6 +1967,80 @@ CPTexturedBackgroundWindowMask
 
         case CPLeftMouseDragged:
         case CPRightMouseDragged:
+            // First, we search for any tracking area requesting CPTrackingEnabledDuringMouseDrag.
+            // At the same time, we update the entered stack.
+            var mouseEnteredStack = [];
```

Comment:
Let's cache some values to avoid method calls in the loop. Add `isKeyWindow = [self isKeyWindow]` to this var block.